### PR TITLE
fix: use last query uuid for custom events

### PR DESCRIPTION
**Pull request by dmbrooke**
[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)

[KIT-2738](https://coveord.atlassian.net/browse/KIT-2738)

There was a mislabelled field for outgoing custom events. It was also suggested that search events may have been sent _**after**_ the custom event (which causes problems since the custom event depends on the last search query UUID), but after doing some digging, I don't see how this could happen unless the implementer themselves were doing thing incorrectly. So for now, I'm just doing the quickest fix to ensure the custom events are properly linked to their respective search events.

TLDR; Custom events were using incorrect field to link to the last query. Should be good to go now.

[KIT-2738]: https://coveord.atlassian.net/browse/KIT-2738?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by louis-bompart — APPROVED** [review]

**Review by mrrajamanickam-coveo — APPROVED** [review]
As discussed, would be good if you can test this manually by triggering the custom events from dev and checking for the events in Admin console. Would be also very helpful if you can post screenshots showing the fixes in admin console analytics in the corresponding discuss.
